src/odemis/driver/andorshrk.py

```diff
@@ -328,6 +333,9 @@ def __init__(self, name, role, device, camera=None, accessory=None,
         dependencies (None or dict str -> HwComponent): if the key starts with
           "led_prot", it will set the .protection to True any time that the
           slit leds could be turned on.
+        :param check_move: Name of the axis -> bool: If True (default), after move, raise an error if
```

Comment:
While your adjusting this docstring, could you fix the order and formatting of the whole docstring? And remove the `inverted` argument from the docstring.

Reply:
Well, I'd rather do it separately. I already have another change (iris support) in the make, that will add yet another parameter. I can clean up the docstring when I create a PR for this change.
As for the `inverted` argument, normally every Actuator accepts it (via the `**kwargs`), but it doesn't work here. So it has to stay documented that this is *not* supported.

Reply:
Okay! Although I would then add the part about the `inverted` argument all the way at the end as a note. I don't use the `inverted` argument often in other Actuators, so to me this description made no sense.